src/invert/laplace/impls/cyclic/cyclic_laplace.cxx

```diff
@@ -482,7 +482,7 @@ void LaplaceCyclic ::verify_solution(const Matrix<dcomplex>& a_ver,
                                      const Matrix<dcomplex>& b_ver,
                                      const Matrix<dcomplex>& c_ver,
                                      const Matrix<dcomplex>& r_ver,
-                                     const Matrix<dcomplex>& x_sol, const int nsys) {
+                                     const Matrix<dcomplex>& x_sol, int nsys) {
```

Comment:
Keeping the value const in the cxx file is perfectly fine, and probably even nicer.

The `const` doesn't have to be in the interface file (but it is no error to have it there), as it does not change the interface.

Removing the const from the implementation changes however the implementation, as `nsys` can now be (accidentally) changed.